src/clr-icons/interfaces/icon-alias.ts

```diff
@@ -1,6 +0,0 @@
-/*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
- * This software is released under MIT license.
- * The full license information can be found in LICENSE in the root directory of this project.
- */
-export interface IconAlias { [key: string]: string[]; }
```

Comment:
I think `IconAlias` and `IconTemplate` interfaces are removed because they aren't being used anymore. Is that correct?

Reply:
They are moved to `icon-interfaces`.